api_docs/incoming-webhooks-overview.md

```diff
@@ -155,3 +155,32 @@ below are for a webhook named `MyWebHook`.
   testing with live data from the service you're integrating and can help you
   spot why something isn't working or if the service is using custom HTTP
   headers.
+
+## URLs
+
+The base URL for an incoming webhook integration bot is
+`{{ api_url }}/v1/external/integration_name?api_key=abcdefgh` where
+`integration_name` is the name of the specific webhook integration and
+`api_key` is the API key of the bot created by the user for the
+integration.
```

Comment:
This is a bit confusing because `integration_name` and `api_key` are implicitly given two different conventions for how to replace things in the example: the reader is expected to replace the literal text "integration_name", but the value that comes after the literal text "api_key". I think both of those are pretty readable conventions, but the juxtaposition is odd.

Here's one way to use a single convention for both:

```suggestion
`{{ api_url }}/v1/external/INTEGRATION_NAME?api_key=API_KEY` where
`INTEGRATION_NAME` is the name of the specific webhook integration and
`API_KEY` is the API key of the bot created by the user for the
integration.
```